pkg/apis/intevents/v1/pullsubscription_defaults.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"context"
+	"time"
+
+	"knative.dev/pkg/apis"
+
+	duck "github.com/google/knative-gcp/pkg/apis/duck"
+	"knative.dev/pkg/ptr"
+)
+
+const (
+	defaultRetentionDuration = 7 * 24 * time.Hour
+	defaultAckDeadline       = 30 * time.Second
+)
```

Comment:
Can we move this constants to pkg/apis/intevents/register.go? Similar to what we did for the duck ones, so that we avoid duplicating this. 
And make v1alpha1, v1beta1 and this use those?